addons/hr_recruitment_survey/wizard/__init__.py

```diff
@@ -1,3 +1,3 @@
 # -*- coding: utf-8 -*-
 
-from . import survey_invite
+from . import recruitment_survey_invite
```

Comment:
Small detail but all the existing recruitment models are named HrRecruitmentSomething, though I see the wizards in `hr_recruitment` don't follow this convention :sweat_smile: 
```suggestion
from . import hr_recruitment_survey_invite
```